Test/PurchaseTest.cs

```diff
@@ -29,6 +29,116 @@ public void InvoicePurchase()
             Assert.Equal(collection.ChargeInvoice.Address.Company, "Acme Software");
         }
 
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void InvoicePurchaseWithRampPlan()
+        {
+            var account = CreateNewAccountWithBillingInfo();
+            var currency = "USD";
+            var plan = CreateNewRampPlan(3);
+            PlansToDeactivateOnDispose.Add(plan);
+
+            var purchase = new Purchase(account.AccountCode, currency);
+            purchase.Account.BillingInfo = account.BillingInfo;
+
+            var sub = new Subscription(plan.PlanCode);
+            purchase.Subscriptions.Add(sub);
+
+            var collection = Purchase.Invoice(purchase);
+            int firstRampUnitAmount = plan.RampIntervals[0].Currencies[0].UnitAmountInCents;
```

Comment:
For my own understanding, shouldn't `var` be fine here, since the type is easily inferred?

Reply:
it could be "fine" there.  But better to be explicit when you can.

Reply:
Typically when its the base types like `int` or `string` or `decimal`, youre explicit about it in c# when you can be